package.json

```diff
@@ -803,6 +803,33 @@
           "group": "navigation@1"
         }
       ],
+      "editor/context": [
```

Comment:
This code seem indented with 4 spaces or something, please be consistent.

Before merging this I'll have to check if this actually works, but also if the context menu doesn't get too crowded with all these extra commands.

Reply:
I did not found what you say about the indentation. The modified code is working. I tested here.

Reply:
You're using tabs instead of spaces. It may look the same, but it's not. 

Actually it doesn't look the same here on github, you can see that your code is more indented than the rest.

Reply:
I've just tested this and it's working, but I don't think all our commands in the context menu should be prefixed with "Todo: ", they should be prefixed in the command palette, but I think it would look better without the prefix in the context menu.

Reply:
Do you want me to modify it again ?

The context menu does not get crowded because is only shown when you have focus in a Todo file

Reply:
As far as I know you can not duplicate the commands in the package.json file, so I would need to review the code, and for now I do not have the time to do it.

I can update the package.son doing the first 2 changes if you wish